src/Umbraco.Web/Composing/CompositionExtensions/WebMappingProfiles.cs

```diff
@@ -1,48 +1,35 @@
-﻿using AutoMapper;
-using Umbraco.Core;
+﻿using Umbraco.Core;
 using Umbraco.Core.Composing;
-using Umbraco.Core.Models;
-using Umbraco.Web.Models.ContentEditing;
+using Umbraco.Core.Mapping;
 using Umbraco.Web.Models.Mapping;
-using Umbraco.Web.Trees;
 
 namespace Umbraco.Web.Composing.CompositionExtensions
 {
     public static class WebMappingProfiles
     {
         public static Composition ComposeWebMappingProfiles(this Composition composition)
         {
-            //register the profiles
-            composition.Register<Profile, AuditMapperProfile>();
-            composition.Register<Profile, CodeFileMapperProfile>();
-            composition.Register<Profile, ContentMapperProfile>();
-            composition.Register<Profile, ContentPropertyMapperProfile>();
-            composition.Register<Profile, ContentTypeMapperProfile>();
-            composition.Register<Profile, DataTypeMapperProfile>();
-            composition.Register<Profile, EntityMapperProfile>();
-            composition.Register<Profile, DictionaryMapperProfile>();
-            composition.Register<Profile, MacroMapperProfile>();
-            composition.Register<Profile, MediaMapperProfile>();
-            composition.Register<Profile, MemberMapperProfile>();
-            composition.Register<Profile, RedirectUrlMapperProfile>();
-            composition.Register<Profile, RelationMapperProfile>();
-            composition.Register<Profile, SectionMapperProfile>();
-            composition.Register<Profile, TagMapperProfile>();
-            composition.Register<Profile, TemplateMapperProfile>();
-            composition.Register<Profile, UserMapperProfile>();
-            composition.Register<Profile, LanguageMapperProfile>();
+            composition.WithCollectionBuilder<MapperProfileCollectionBuilder>()
+                .Append<AuditMapperProfile>()
+                .Append<CodeFileMapperProfile>()
+                .Append<ContentMapperProfile>()
+                .Append<ContentPropertyMapperProfile>()
+                .Append<ContentTypeMapperProfile>()
+                .Append<DataTypeMapperProfile>()
+                .Append<EntityMapperProfile>()
+                .Append<DictionaryMapperProfile>()
+                .Append<MacroMapperProfile>()
+                .Append<MediaMapperProfile>()
+                .Append<MemberMapperProfile>()
+                .Append<RedirectUrlMapperProfile>()
+                .Append<RelationMapperProfile>()
+                .Append<SectionMapperProfile>()
+                .Append<TagMapperProfile>()
+                .Append<TemplateMapperProfile>()
+                .Append<UserMapperProfile>()
+                .Append<LanguageMapperProfile>();
 
-            //register any resolvers, etc.. that the profiles use
-            composition.Register<ContentUrlResolver>();
```

Comment:
All these Resolvers seem to be gone - so what do we do to replace these?

Reply:
we don't "resolve" anymore, our mappers explicitely do what the resolvers were doing in the past